sfe updates (#146)
codatio/codat-docs: docs/sync-for-expenses/sync-process/expense-transactions.md

          },
          "accountRef": {
            "id": "08ca1c6e-0374-11ed-b939-0242ac120002"
          },
          "trackingRefs": [
            {
              "id": "08ca1c6e-0374-11ed-b939-0242ac120002"
            }
          ]
        }
      ],
      "notes": "Amazon UK | Online Purchase | Order 123XX45"
    }
  ]
}
```

## Transaction Types

The way Codat handles, maps and processes a transaction is based on the specified `type` of the transaction.

<ul className="card-container col-2">
  <li className="card">
    <div class="header">
      <h3>payment</h3>
    </div>
    <p>
      Is used to represent any spend that takes place on the account and interest on credit purchases. 
    </p>
  </li>

  <li className="card">
    <div class="header">
      <h3>refund</h3>
    </div>
    <p>
      Can be used to represent any refunds and returns on an original transaction.
    </p>
  </li>

  <li className="card">
    <div class="header">
      <h3>reward</h3>
    </div>
    <p>
      Can be used to represent reward redemptions such as cashback.
    </p>
  </li>
<li className="card">
    <div class="header">
      <h3>chargeback</h3>
    </div>
    <p>

>       Is similar, to a refund in behaviour and represents a return of transaction/payment sum which may have been disputed.

Reviewer: ```suggestion
      Is similar to a refund in behavior and represents a return of transaction or payment sum which may have been disputed.
```

    </p>
  </li>

  <li className="card">
    <div class="header">
      <h3>transferIn</h3>
    </div>
    <p>
      A transfer that decreases the balance of the credit card account or increases the balance of a bank account. </p>
<p>Can be used to represent a top up of debit card account, a pay down of a credit card account or a balance transfer to another credit card.</p>
    
  </li>

  <li className="card">
    <div class="header">
      <h3>transferOut</h3>
    </div>
    <p>
      A transfer that increases the balance of the credit account or decreases the balance of a bank account.</p>
<p> Can be used to represent cash withdrawals or balance transfer to another credit card.</p>
    
  </li>
  <li className="card">
    <div class="header">
      <h3>adjustmentIn</h3>
    </div>
    <p>
      An adjustment that decreases the balance of the credit account or increases the balance of a bank account. </p>

>       <p>Can be used to represent write-off's & transaction adjustments such as foreign exchange adjustments. 

Reviewer: ```suggestion
      <p>Can be used to represent write-offs & transaction adjustments such as foreign exchange adjustments. 
```

    </p>
  </li>

  <li className="card">
    <div class="header">
      <h3>adjustmentOut</h3>
    </div>
    <p>
      An adjustment that increases the balance of the credit account or decreases the balance of a bank account. </p>

> <p>Can be used to represent write-off's & transaction adjustments such as foreign exchange adjustments. 

Reviewer: ```suggestion
<p>Can be used to represent write-offs & transaction adjustments such as foreign exchange adjustments. 
```
